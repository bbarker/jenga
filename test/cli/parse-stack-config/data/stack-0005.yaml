resolver: lts-11.13
flags: {}
extra-deps: []
extra-package-dbs: []

packages:
  - '.'

  - location:
      git: https://github.com/well-typed/cborg
      commit: c7db82bfd93923f5b08ed51a4cd53e30bd445924
    subdirs:
    - cborg
    extra-dep: true

  - location:
      git: https://github.com/input-output-hk/engine.io.git
      commit: d3c55f51bb81cee7d0d551de930ce65fe7d76756
    extra-dep: true
    subdirs:
      - socket-io
      - engine-io
      - engine-io-wai
