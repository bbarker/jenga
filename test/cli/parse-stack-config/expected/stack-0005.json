{
    "packages": [
        ".",
        {
            "location": {
                "git": "https://github.com/well-typed/cborg",
                "commit": "c7db82bfd93923f5b08ed51a4cd53e30bd445924"
            },
            "subdirs": [
                "cborg"
            ],
            "extra-dep": true
        },
        {
            "location": {
                "git": "https://github.com/input-output-hk/engine.io.git",
                "commit": "d3c55f51bb81cee7d0d551de930ce65fe7d76756"
            },
            "subdirs": [
                "socket-io",
                "engine-io",
                "engine-io-wai"
            ],
            "extra-dep": true
        }
    ],
    "extra-deps": [],
    "resolver": "lts-11.13"
}
